docs/sql-performance-tuning.md

```diff
@@ -228,6 +228,8 @@ The "REPARTITION_BY_RANGE" hint must have column names and a partition number is
     SELECT /*+ REPARTITION */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
+    SELECT /*+ REPARTITION_BY_AQE */ * FROM t
```

Comment:
I just saw this. What's the diff between ` /*+ REPARTITION */ ` and `/*+ REPARTITION_BY_AQE */`?

Reply:
currently they are same, but I prefer the later which is more safer with AQE

Reply:
Can we disallow `/*+ REPARTITION_BY_AQE */` case?